sites/partners/src/components/listings/PaperListingDetails/sections/DetailApplicationDates.tsx

```diff
@@ -87,47 +87,56 @@ const DetailApplicationDates = () => {
         )}
 
         <Drawer
-          open={!!drawer}
-          title={t("listings.sections.openHouse")}
-          ariaDescription={t("listings.unit.title")}
+          isOpen={!!drawer}
+          ariaLabelledBy="detail-application-dates-drawer-header"
+          ariaDescribedBy="detail-application-dates-drawer-description"
           onClose={() => setDrawer(null)}
         >
-          <Card spacing="lg" className="spacer-section">
-            <Card.Section>
-              <Grid className="grid-inset-section">
-                <Grid.Row columns={3}>
-                  <FieldValue id="drawer.startTime.date" label={t("t.date")}>
-                    {drawer?.startTime && getDetailFieldDate(drawer.startTime)}
-                  </FieldValue>
-                  <FieldValue id="drawer.startTime.time" label={t("t.startTime")}>
-                    {getDetailFieldTime(drawer?.startTime)}
-                  </FieldValue>
-                  <FieldValue id="drawer.endTime.time" label={t("t.endTime")}>
-                    {drawer?.endTime && getDetailFieldTime(drawer?.endTime)}
-                  </FieldValue>
-                  <FieldValue id="drawer.url" label={t("t.url")}>
-                    {drawer?.url ? (
-                      <Link className="mx-0 my-0" href={drawer.url}>
-                        {drawer?.label ?? t("t.url")}
-                      </Link>
-                    ) : (
-                      t("t.n/a")
-                    )}
-                  </FieldValue>
-                  <FieldValue
-                    id="events.openHouseNotes"
-                    label={t("listings.events.openHouseNotes")}
-                  >
-                    {drawer?.note || t("t.n/a")}
-                  </FieldValue>
-                </Grid.Row>
-              </Grid>
-            </Card.Section>
-          </Card>
-
-          <Button variant="primary" onClick={() => setDrawer(null)}>
-            {t("t.done")}
-          </Button>
+          <Drawer.Header id="detail-application-dates-drawer-header">
+            {t("listings.sections.openHouse")}
+          </Drawer.Header>
+          <Drawer.Content>
+            <p id="detail-application-dates-drawer-description" className="sr-only">
```

Comment:
What's the purpose of this instead of using the open house string?

Reply:
I'm not sure, I was just trying to replicate the pre-Seeds behavior. We could probably just remove it?